src/cljs/metafacture_playground/db.cljs

```diff
@@ -1,29 +1,39 @@
 (ns metafacture-playground.db)
 
 (def default-db
-  {:input-fields {:data {:key-count 0
-                         :content nil
-                         :collapsed? false
-                         :width nil
-                         :disabled? true}
-                  :flux {:key-count 0
-                         :content nil
-                         :collapsed? false
-                         :width nil}
-                  :fix {:key-count 0
-                        :content nil
-                        :disabled? true}
-                  :morph {:key-count 0
-                          :content nil
-                          :disabled? true}
-                  :switch {:collapsed? false
-                           :active :fix
-                           :width nil}}
+  {:editors {:data {:key-count 0
+                    :content nil
+                    :collapsed? false
+                    :disabled? true
+                    :label "\"inputFile\"-content"
```

Comment:
I like the italics variant you showed in https://github.com/metafacture/metafacture-playground/issues/114#issuecomment-1700567788 more than the double quotes (cc @TobiasNx).

Reply:
A third idea: remove the double quotes (without italic)? 🤷 @TobiasNx @fsteeg